src/app/qgssnappingwidget.cpp

```diff
@@ -173,6 +175,23 @@ QgsSnappingWidget::QgsSnappingWidget( QgsProject *project, QgsMapCanvas *canvas,
   mToleranceSpinBox->setObjectName( QStringLiteral( "SnappingToleranceSpinBox" ) );
   connect( mToleranceSpinBox, static_cast < void ( QDoubleSpinBox::* )( double ) > ( &QDoubleSpinBox::valueChanged ), this, &QgsSnappingWidget::changeTolerance );
 
+  mMinScaleWidget = new QgsScaleWidget();
+  mMinScaleWidget->setToolTip( tr( "Min scale on which snapping is enabled" ) );
```

Comment:
```suggestion
  mMinScaleWidget->setToolTip( tr( "Minimum scale from which snapping is enabled" ) );
```
maybe some native speaker can come up with something better

Reply:
if no ones comes up with something better, I would recommend to accept this one

Reply:
Yes, I went with it.